docs/guides/advanced/ServerRendering.md

```diff
@@ -53,7 +53,7 @@ render(<Router history={history} routes={routes} />, mountNode)
 You need to do
 
 ```js
-match({ routes, location }, (error, redirectLocation, renderProps) => {
-  render(<Router {...renderProps} history={history} />, mountNode)
+match({ history, routes }, (error, redirectLocation, renderProps) => {
```

Comment:
I'm actually less bothered by this specific case – both async routes and SSR are relatively advanced, so the documentation around the intersection of the two doesn't need to target beginners.

It's not just "we don't have to pass through our history" in this case though – it's actually necessary that both `match` and `<Router>` use the same `history`, since the transition manager (that we want to re-use) is already bound to a history.